src/components/Header.tsx

```diff
@@ -1,35 +1,81 @@
-import React from 'react';
-import logoHouseText from '@/assets/logoHouseText.png';
+'use client';
+import React, { useState } from 'react';
 import Image from 'next/image';
+import logoHouseText from '@/assets/logoHouseText.png';
 import NavLink from './NavLink';
 import { Button } from './ui/button';
+import { Menu } from 'lucide-react';
 
 const Header = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
   return (
-    <section className='flex flex-row items-center justify-between px-5'>
-      {/* Navbar */}
-      <div className='flex flex-row gap-3 items-center'>
+    <header className='w-full flex flex-col md:px-[50px] relative'>
+      {/* Top section with logo, nav links, and hamburger menu */}
+      <div className='flex flex-row items-center justify-between w-full'>
         {/* Logo */}
-        <Image
-          src={logoHouseText}
-          width={120}
-          height={120}
-          priority={true}
-          alt='Logo House Text'
-        />
-        {/* Nav Links */}
-        <NavLink />
-      </div>
-      {/* Sign up + Log in */}
-      <div className='flex gap-3'>
-        <Button className='bg-white text-primaryDark ring-1 ring-primaryDark hover:bg-slate-400'>
-          Đăng ký
-        </Button>
-        <Button className='bg-primaryLight  text-white   hover:bg-primaryDark'>
-          Đăng nhập
-        </Button>
+        <div className='flex items-center'>
+          <Image
+            src={logoHouseText}
+            width={180}
+            height={180}
```

Comment:
![image](https://github.com/user-attachments/assets/142d5065-3eac-47bc-9761-2bba82d2f0f3)
 can you make it more suitable for mobile ? 
 
 made by fill in like this 
![image](https://github.com/user-attachments/assets/53f05e42-f37e-4339-9ef0-19b5c5c88f7d)

Reply:
I usually test with devices such as Iphone 12 Pro and XR device so I think it won't be necessary and I asked Long before and he agreed with me that we don't need to responsive on tablet devices.